.gitattributes

```diff
@@ -1,4 +1,4 @@
 *.sh text eol=lf
 ./public/static/dist/* binary
 ./public/package-lock.json binary
-graphql/*_resolver.go -linguist-generated
+graphql/*_resolver.go linguist-detectable linguist-generated=false linguist-language=go
```

Comment:
I'm not sure that `=false` is valid linguist syntax. In [this](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#detectable) example, they demonstrate unmarking a path as detectable with `-linguist-detectable` which suggests that `-` is how negation is done.

Reply:
Wow nice!! @khelif96 should we raise this in the GitHub Support thread?